pygeppetto/model/model_serializer.py

```diff
@@ -1,13 +1,29 @@
 import sys
 
 from pyecore.resources import ResourceSet
-from pyecore.resources.json import JsonOptions
+from pyecore.resources.json import JsonOptions, DefaultObjectMapper
 
 from .geppetto_resource import GeppettoResource
 from .utils.bytesuri import BytesURI
 
 GeppettoModelSerializer = GeppettoSerializer = sys.modules[__name__]
 
+from pyecore.resources.json import DefaultObjectMapper
+
+class SynchableMapper(DefaultObjectMapper):
```

Comment:
With the modification I will perform on PyEcore side, the `SynchableMapper` will change a little bit (not so much)

Reply:
Hi @aranega is that modification you mentioned done now?